examples/asset/multi_asset_sync.rs

```diff
@@ -1,12 +1,10 @@
 //! This example illustrates how to wait for multiple assets to be loaded.
 
+use std::sync::Arc;
 use std::{
     f32::consts::PI,
     ops::Drop,
-    sync::{
-        atomic::{AtomicBool, AtomicU32, Ordering},
-        Arc,
-    },
+    sync::atomic::{AtomicBool, AtomicU32, Ordering},
```

Comment:
why move `std::sync::Arc` outside of the other `std` imports?

Reply:
This was not a deliberate change, just a consequence of the `rustfmt` process used to address `clippy`. I'll update the PR to re-merge any items that got split out in examples.